app/controllers/host_initiator_controller.rb

```diff
@@ -0,0 +1,47 @@
+class  HostInitiatorController < ApplicationController
```

Comment:
I think rubocop is complaining about the extra space here...

Reply:
fixed